src/pages/index.md

```diff
@@ -2,8 +2,43 @@
 tags: ["intro"]
 ---
 
-CosmWasm is a smart contract platform focusing on security, performance, and
-interoperability. It is the only smart contracting platform for public
-blockchains with heavy adoption outside of the EVM world.
+The [Cosmos Wasm](https://github.com/CosmWasm/wasmd/tree/main/x/wasm), the root of CosmWasm, is a
+[CosmosSDK](https://docs.cosmos.network/) module enabling Smart Contracts to execute on the Wasm virtual machine.
+It is a bridge between Cosmos chain and the [WasmWM](https://github.com/CosmWasm/wasmvm) executing the Smart Contracts.
+CosmWasm itself is the whole ecosystem built around it with a missing to make Smart Contracts development easy and reliable.
+The focuses of the CosmWasm platform are security, performance, and interoperability. CosmWasm is the only Smart Contracts
+platform for public blockchains that has been heavily adopted outside the EVM world.
```

Comment:
uhhh, that's a bold claim. There is also SVM and Move

Reply:
Yea, I would get rid of that. I kinda forwarded it as it was here before, but I also think it is kinda bold.